docs/getting-started/_index.en.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started with Chalk
+
+Welcome to Chalk, the open-source observability tool that provides full lifecycle visibility for
+your software by Crash Override. Whether you're looking to track software from development to
+production, enhance your supply chain security, or create a comprehensive application inventory,
+Chalk offers a powerful yet straightforward approach to software metadata management.
+
+This Getting Started guide will walk you through the essentials of Chalk, from installation to
+example integrations with your CI/CD pipelines. By following these sections in order, you will gain
+a comprehensive understanding of Chalk's core capabilities and be ready to implement it in your own
+environment.
+
+## Your Learning Path
+
+We recommend working through these sections in sequence, as each builds upon the concepts introduced
+in the previous ones:
+
+### 1. [Installation Guide](./installation.en.md)
+
+Begin your journey by installing Chalk on your system. This section covers both downloading
+pre-built binaries (the simplest approach) and building Chalk from source. You'll learn how to set
+up Chalk so it's ready to use in your local environment or deployment pipelines.
+
+### 2. [Chalk Basics](./chalk-basics.en.md)
+
+Once Chalk is installed, this section will introduce you to the fundamental
+operations: chalk marking binaries and container images, extracting chalk marks to view embedded
+metadata, and configuring basic reporting. These simple examples will help you understand Chalk's
+core functionality and provide a foundation for more advanced usage.
+
+### 3. [Using Chalk Exec Reports](./chalk-exec-reports.en.md)
+
+After mastering the basics, dive into Chalk's powerful execution reporting capabilities. Learn how
+to capture runtime information about your applications, set up heartbeat monitoring for continuous
+observation, and customize reporting to focus on the data most relevant to your needs. This section
+bridges the gap between build-time metadata and runtime behavior.
+
+### 4. [Integrating with CI/CD](./integrating-with-ci-cd.en.md)
+
+With a solid understanding of Chalk's capabilities, you will be ready to implement it in your
+development workflows. This section explores how to integrate Chalk into popular CI/CD platforms
+like GitHub Actions, GitLab CI, Jenkins, CircleCI, and Azure DevOps. You'll learn best practices
+for automating metadata collection and ensuring consistent marking across your artifacts.
+
+## What Comes Next?
+
+After completing these initial sections, you will have a strong foundation in Chalk's core
+functionality. From there, you can explore more advanced topics based on your specific needs:
+
+- Customizing configurations for specialized use cases
+- Setting up digital signing and attestation
+- Implementing SBOM generation and security scanning
```

Comment:
@miki725 i think this is what we're currently missing. we can update this with links to your new docs.